### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by settermjd**
This PR corrects a series of small issues that cause the build process to throw warnings and errors.

### CODE REVIEW

File: modules/ROOT/pages/configuration/user/user_provisioning_api.adoc [hunk 1]
```diff
@@ -147,7 +147,7 @@ sending a Basic HTTP Authorization header.
 [cols=",,",options="header",]
 |==============================================================
 |Request Path |Method |Content Type
-|`Syntax: ocs/v1.php/cloud/users/{userid}` |`GET` |`text/plain`
+| `Syntax: ocs/v1.php/cloud/users/\{userid\}` | `GET` | `text/plain`
```
Comment (mmattel) [inline modules/ROOT/pages/configuration/user/user_provisioning_api.adoc]: You are using a blank between `|`` and the next character, this is good and improves readability !
Also used in other tables like occ commands.
To be consistent, you should do that on all following tables too...

**Comment by mmattel** [comment]
Lookes good, a small thing to improve

File: modules/ROOT/pages/configuration/user/user_provisioning_api.adoc (hunk 1 above)
Comment (settermjd) [reply modules/ROOT/pages/configuration/user/user_provisioning_api.adoc]: That somehow slipped in. It was meant to only be a part of #143.

Comment (mmattel) [reply modules/ROOT/pages/configuration/user/user_provisioning_api.adoc]: without double checking, is this then addressed in #143 ?
If not pls add there as this PR is mergen now

Comment (settermjd) [reply modules/ROOT/pages/configuration/user/user_provisioning_api.adoc]: Yep, the change is there. As I said, this crept through, by accident.

**Pull request merged**